doc/refvariable.rst

```diff
@@ -95,6 +95,23 @@ This form of constructing variables is not considered idiomatic JuMP code.
     The reference to the first set of variables has been lost, although they will remain
     in the model.
 
+Semidefinite variables
+^^^^^^^^^^^^^^^^^^^^^^
+
+JuMP supports modeling with `semidefinite variables <https://en.wikipedia.org/wiki/Semidefinite_programming>`_. A square symmetric matrix :math:`X` is semidefinite if all eigenvalues are nonnegative; this is typically denoted by :math:`X \succeq 0`. A
+
+    @defVar(m, X[1:3,1:3], SDP)
+
+Note in particular the indexing: 1) exactly two index sets must be specified, 2) they must both be unit ranges starting at 1. Variable bounds have special meaning with the ``SDP`` annotation; for example,
+
+    n = 3
+    @defVar(m, X[1:n,1:n] >= eye(n), SDP)
+
+constrains :math:`X - I \succeq 0`. In particular, this is (almost) equivalent to
```

Comment:
No, but ``X ⪰0`` [is](https://web.stanford.edu/~boyd/papers/pdf/semidef_prog.pdf). My only point being that this would be a more direct translation of something like, e.g. (7). If your suggesting that we remove support for SDP bounds in ``@defVar``, though, I could probably get behind that.